bartok765/galaxy_blizzard_plugin
# Implement `get_local_size`

## Description
This implements Galaxy API feature in a simple way: stats files available in a game's installed directory and sum their sizes.

The function is synchronious and may lasts long time (as the `os.stat` waits for OS), so plugin runs it in separate thread to not block event loop.

## How has this been tested?

 

- Connect plugin
- click on battlenet tab in Galaxy
- change games view from grid to list
- click customization icon -> `Customize list view`
- select `Size on disk`
- wait a few seconds to see size showing up next to installed games

Tested on Windows10 only

## Checklist:

- [x] I have added myself to contributor list in `src/manifest.json` (nick or/and full name)
- [x] I have added myself to contributor list in `LICENSE` file (nick or/and full name)

**New Unit Tests**
- [x] I have added/modified unit tests to cover my change
